libsuperiormysqlpp.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@PREFIX@
+prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include
 
 Name: SuperiorMySqlpp
 Description: SuperiorMySqlpp
-Version: @VERSION@
-Libs: -lmysqlclient
+Version: @SuperiorMySqlpp_VERSION@
+Libs: -lmysqlclient -pthread
```

Comment:
I think it wasn't specified here on purpose - setting thread library is not entirely portable (like `-pthreads` on Solaris or `-mthread`).

Reply:
I just followed your comment from earlier, kinda.
https://github.com/seznam/SuperiorMySqlpp/pull/129#discussion_r506574204
So propagate it via var instead?